### PR TITLE
LinalgRefactor-SGVector-sum

### PR DESCRIPTION
**Pull request by OXPHOS**

### CODE REVIEW

File: src/shogun/mathematics/linalg/LinalgBackendBase.h [hunk 1]
```diff
@@ -50,6 +50,33 @@ class LinalgBackendBase
 {
 public:
 	/**
+	 * Wrapper method of add operation the operation C = alpha*A + beta*B.
```
Comment (karlnapf) [inline src/shogun/mathematics/linalg/LinalgBackendBase.h]: add again?

File: src/shogun/mathematics/linalg/LinalgBackendEigen.h [hunk 2]
```diff
@@ -44,6 +44,29 @@ namespace shogun
 class LinalgBackendEigen : public LinalgBackendBase
 {
 public:
+	/** Implementation of @see LinalgBackendBase::add */
+	#define BACKEND_GENERIC_ADD(Type) \
+	virtual SGVector<Type> add(const SGVector<Type>& a, const SGVector<Type>& b, Type alpha, Type beta) const \
```
Comment (karlnapf) [inline src/shogun/mathematics/linalg/LinalgBackendEigen.h]: add?

**Comment by karlnapf** [comment]
Remove the add and I am fine to merge

**Comment by karlnapf** [comment]
close?

**Comment by OXPHOS** [comment]
@karlnapf by mistake and then i forced updated..check #3343

**Comment by karlnapf** [comment]
I see, no need to re-open PRs if you force updates btw

**Pull request closed without merging**